CONTRIBUTING.md

```diff
@@ -418,3 +418,9 @@ potentially interested reviewers.
 ```
 git send-email --to=git@vger.kernel.org --cc=<email1> --cc=<email2> [dir with patches]/*.patch
 ```
+
+To submit a nth version patch (say version 3):
+
+```
+git send-email --to=git@vger.kernel.org --cc=<email1> --cc=<email2> --in-reply-to=<the message id of cover letter of patch v2> [dir with patches]/*.patch
```

Comment:
I was also going to suggest adding the `-vn` option i.e. `-v3` in this example.

Reply:
PR? 😄

Reply:
🤦‍♂️ Doh moment, it's `format-patch` not `send-email` that takes the `-vn` option, and thats alreay covered in #L404 [wanted to put in link there but can't see how to do it on GiHub for .md files which are rendered without line numbers and the raw view doesn't have then either]

Reply:
Yeah, `format-patch` for `-vn` is already done. I wanted to add an example for submitting a `vn` patch using `send-email` as I got confused earlier.

Reply:
> it's format-patch not send-email that takes the -vn option

Note: you can apparently also call `send-email` with `format-patch` options instead of a directory or some files.